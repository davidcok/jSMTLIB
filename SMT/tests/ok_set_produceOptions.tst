; checks the set and get of produce options
(set-option :produce-proofs true)
(get-option :produce-proofs)
(set-option :produce-proofs false)
(get-option :produce-proofs)
(set-option :produce-unsat-cores true)
(get-option :produce-unsat-cores)
(set-option :produce-unsat-cores false)
(get-option :produce-unsat-cores)
(set-option :produce-assignments true)
(get-option :produce-assignments)
(set-option :produce-assignments false)
(get-option :produce-assignments)
(set-option :produce-models true)
(get-option :produce-models)
(set-option :produce-models false)
(get-option :produce-models)
