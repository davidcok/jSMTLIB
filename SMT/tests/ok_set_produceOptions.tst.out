unsupported
false
unsupported
false
unsupported
false
unsupported
false
unsupported
false
unsupported
false
unsupported
false
unsupported
false
